content/topics/intro-to-typescript/_index.md

```diff
@@ -5,11 +5,28 @@ ready: true
 title: Introduction to Typescript
 ---
 
+### What is Typescript
+
+- TypeScript is an extension of JavaScript that adds types as well as other language features.
+- Typescript was created to make it easier to write code that is compatible with the JavaScript language.
+- As its name implies, adding types to JavaScript is the primary purpose of TypeScript, and it is the reason that people use it today.
+
+#### The TypeScript Compiler
+
+- TypeScript must be compiled, or transpiled, as it is frequently termed, into JavaScript for execution. TypeScript gets compiled to JavaScript by the TypeScript Compiler.
```

Comment:
TypeScript gets transpiled - transpilers are subsets of compilers, you perhaps would wanna be direct and use a subset rather